spec/factories/challenge.rb

```diff
@@ -50,6 +50,20 @@
     #   create_list(:phase, 1, challenge: challenge)
     # end
 
+    after(:create) do |challenge, _evaluator|
+      # Create challenge_manager
+      create(:challenge_manager, challenge: challenge, user: challenge.user)
+
+      # Create phases
+      if challenge.is_multi_phase && challenge.phases.empty?
```

Comment:
not sure if this matters much but if the challenge is multi-phase and has phases defined, I think this will create another phase